packages/codegen-ui-react/lib/react-component-with-children-renderer.ts

```diff
@@ -58,9 +59,14 @@ export class ReactComponentWithChildrenRenderer<TPropIn> extends ComponentWithCh
   }
 
   protected renderOpeningElement(): JsxOpeningElement {
-    const attributes = Object.entries(this.component.properties).map(([key, value]) =>
-      buildOpeningElementAttributes(value, key),
-    );
+    const attributes = Object.entries(this.component.properties)
+      .map(([key, value]) => buildOpeningElementAttributes(value, key))
+      .concat(
+        Object.entries(this.component.eventProperties || {}).map(([key, value]) =>
+          // pass as bindingProperties to reuse logic
+          buildOpeningElementAttributes({ bindingProperties: { property: value } }, mapGenericEventToReact(key)),
+        ),
+      );
```

Comment:
Shared code between the child and standard properties. Not critical in this PR, but we should start looking at how to normalize these two if possible.

Reply:
Additionally, this logic is slightly opaque, unless you're cached into event handling specifically. May be more readable to generate bindingProperties, generate eventProperties, then return the concat of those two named vars (no logical change).